crates/opencascade/src/primitives.rs

```diff
@@ -265,18 +222,21 @@ impl Wire {
     }
 
     pub fn translate(&mut self, offset: DVec3) {
-        self.transform(offset, dvec3(1.0, 0.0, 0.0), 0.0);
+        self.transform(offset, dvec3(1.0, 0.0, 0.0), 0.degrees());
     }
 
-    pub fn transform(&mut self, translation: DVec3, rotation_axis: DVec3, angle: f64) {
-        let angle = angle * std::f64::consts::PI / 180.0;
-
+    pub fn transform(
+        &mut self,
+        translation: DVec3,
+        rotation_axis: DVec3,
+        angle: impl Into<Radians>,
+    ) {
         let mut transform = ffi::new_transform();
         let rotation_axis_vec =
             ffi::gp_Ax1_ctor(&make_point(DVec3::ZERO), &make_dir(rotation_axis));
         let translation_vec = make_vec(translation);
 
-        transform.pin_mut().SetRotation(&rotation_axis_vec, angle);
+        transform.pin_mut().SetRotation(&rotation_axis_vec, angle.into().into());
```

Comment:
Ohhh I love this one. Let me add this, as I think it solves my biggest issue with the PR :)

Reply:
Hmm hold on, I answered too quick 😄.

I don't think this one quite works as it would make degrees incorrect, right?

```rust
assert_eq!((PI).radians().value(), PI); //correct
assert_eq!(180.degrees().value,  PI); //Incorrect, should be 180
```

We could have `value_degrees()` and `value_radians()` but that would sort of defeat the gain.

Reply:
Ah true. Damn. :-|